debian/patches/series

```diff
@@ -1,2 +1 @@
-installed-tests-Install-test-keyring2-to-the-right-place.patch
-testlibrary-Call-g_assert_no_error-first.patch
+tests-Isolate-tests-from-real-home-directory-more-thoroug.patch
```

Comment:
You didn't actually add this patch, but I guess it doesn't matter since it gets dropped in 0.9.8.